sphinx/registry.py

```diff
@@ -461,6 +466,41 @@ def get_envversion(self, app: "Sphinx") -> Dict[str, str]:
         envversion['sphinx'] = ENV_VERSION
         return envversion
 
+    def create_publisher(self, app: "Sphinx", filename: str) -> Publisher:
```

Comment:
I considers registry is a mere storage of components. So it should not have "features" as possible. So I'd like to move this `sphinx.io` back again.

Reply:
How's the current?

A

Reply:
Thank you for your update. I prefer the idea of `create_publisher()`. +1 for this.